build.sh

```diff
@@ -0,0 +1,17 @@
+# Remove existing node Docker container
+docker rm -f node
+
+# Run Node v8 in Docker container
+docker run -td --name node -v "$PWD":/usr/src/app -w /usr/src/app node:8 || exit
+
+# Install PhantomJS first to avoid issues on install
+docker exec -it node npm install phantomjs-prebuilt@2.1.16 --ignore-scripts || exit
+
+# Install other dependencies
+docker exec -it node npm install || exit 
```

Comment:
I think it would make more sense to just make a Dockerfile that includes these steps when building, so you could just run `docker build .` Is there a benefit to doing it this way?

Reply:
Maybe there is a better way ... it was a quick solution to handle everything in one file.

Reply:
Feel free to improve the solution :-)